orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/providers/gce/GoogleServerGroupCreator.groovy

```diff
@@ -50,18 +50,18 @@ class GoogleServerGroupCreator implements ServerGroupCreator {
       // Bakery ops are keyed off cloudProviderType
       operation.image = deploymentDetails.find { it.cloudProviderType == 'gce' }?.ami
 
-      // Alternatively, FindImage ops distinguish between server groups deployed to different zones.
+      // Alternatively, FindImage ops distinguish between server groups deployed to different regions.
       // This is partially because AWS images are only available regionally.
-      if (!operation.image && stage.context.zone) {
-        operation.image = deploymentDetails.find { it.zone == stage.context.zone }?.imageId
+      if (!operation.image && stage.context.region) {
```

Comment:
This isn't really directly related to the Clone changes, but fixed this while I was in the neighborhood.